invenio_rdm_records/services/pids/providers/__init__.py

```diff
@@ -8,13 +8,13 @@
 """PID Providers module."""
 
 from .base import BaseClient, BasePIDProvider
-from .datacite import DataCiteClient, DataCitePIDProvider
+from .datacite import DOIDataCiteClient, DOIDataCitePIDProvider
```

Comment:
isn't DOI an overdo in the name? IMO it makes it longer and it does not provide much, since datacite is always doi? or are there more possible pid_types for datacite?

Reply:
used due to `ConceptDOI` provider... feel free to change.